src/adapter/src/catalog/storage.rs

```diff
@@ -992,6 +993,24 @@ impl<'a> Transaction<'a> {
         )
     }
 
+    pub fn rename_user_cluster(
+        &mut self,
+        cluster_id: ClusterId,
+        cluster_name: &str,
+        cluster_to_name: &str,
+    ) -> Result<(), Error> {
+        self.rename_cluster(cluster_id, cluster_name, cluster_to_name)
+    }
+
+    pub fn rename_user_cluster_replica(
+        &mut self,
+        replica_id: ReplicaId,
+        replica_name: &QualifiedReplica,
+        replica_to_name: &str,
+    ) -> Result<(), Error> {
+        self.rename_cluster_replica(replica_id, replica_name, replica_to_name)
+    }
```

Comment:
These seem a bit weird, in that they don't differ from the non `user` variants, and don't limit cluster/replica IDs to just the user variants. Can we just not have them, and have callers call `rename_cluster[_replica]` directly?

Reply:
Removed.